### PR TITLE
Update golang to 1.6.3 + set go1.6 images as primary ones

### PR DESCRIPTION
**Pull request by sdurrheimer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
-- `powerpc`, `1.5-powerpc`, `1.5.4-powerpc` ([1.5/powerpc/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.5/powerpc/Dockerfile))
-- `1.6-main`, `1.6.2-main` ([1.6/main/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.6/main/Dockerfile))
-- `1.6-arm`, `1.6.2-arm` ([1.6/arm/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.6/arm/Dockerfile))
-- `1.6-powerpc`, `1.6.2-powerpc` ([1.6/powerpc/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.6/powerpc/Dockerfile))
-- `1.6-mips`, `1.6.2-mips` ([1.6/mips/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.6/mips/Dockerfile))
+- `1.5-main`, `1.5.4-main` ([1.5/main/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.5/main/Dockerfile))
+- `1.5-arm`, `1.5.4-arm` ([1.5/arm/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.5/arm/Dockerfile))
+- `1.5-powerpc`, `1.5.4-powerpc` ([1.5/powerpc/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.5/powerpc/Dockerfile))
```
Comment (grobie) [inline README.md]: Do we still use the 1.5 images?

**Comment by grobie** [comment]
Cool! :+1:

File: README.md (hunk 1 above)
Comment (sdurrheimer) [inline README.md]: No but others people might. I'm in favor of keeping it until `1.7`.

Comment (grobie) [inline README.md]: No objections, but I wonder why? It's not about keeping the images available in the docker hub, but about starting the list with deprecated images. People will be able to continue using them even if we remove them from the readme, right? If you want to keep them on the mage, maybe list them below the 1.6 images?

Comment (sdurrheimer) [inline README.md]: Done. I have moved `1.5` images below `1.6`.

**Comment by sdurrheimer** [comment]
Arf circleci is failing because of the `/` in the branch name. I'm gonna redo the PR.

**Pull request merged**